index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en" data-bs-theme="dark" class="h-100">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      defer
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2"
+      crossorigin="anonymous"
+    ></script>
+    <title>Alien Social | Login</title>
+    <meta name="description" content="alien something">
+    <link rel="stylesheet" href="/dist/css/index.css" />
+  </head>
+  <body class="d-flex flex-column h-100">
+      <header>
+        <nav class="navbar navbar-expand-lg bg-body-tertiary rounded" aria-label="navigation">
+          <div class="container-fluid">
+            <a class="navbar-brand mt-3" href="/">👽</a>
+            <button
+              class="navbar-toggler"
+              type="button"
+              data-bs-toggle="collapse"
+              data-bs-target="#navbarsExample09"
+              aria-controls="navbarsExample09"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarsExample09">
+              <ul class="navbar-nav mt-3 me-auto mb-2 mb-lg-0 nav nav-pills">
+                <li class="nav-item">
+                  <a class="nav-link active" aria-current="login" href="/">Login</a>
+                </li>
+                <!-- <li class="nav-item">
+                  <a class="nav-link" href="/posts/index.html">Posts</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="/profile/index.html">Profile</a>
+                </li> -->
+              </ul>
+              <!-- <form role="search">
+                <input class="form-control" type="search" placeholder="Search" aria-label="Search" />
+              </form> -->
+            </div>
+          </div>
+        </nav>
+      </header>
+    <main class="flex-shrink-0">
+      <section class="h-100 gradient-custom-2">
```

Comment:
Indentation is weird, please run a formatter over your project. You can install `prettier` and add this to package.json:

```
npm i -D prettier
```

```json
{
  "scripts": {
    "format": "prettier --write --ext '.html' ."
  }
}
```

`npm run format`

Reply:
I have used prettier all the time, also installed the script.